iroha_client_no_std/src/client.rs

```diff
@@ -0,0 +1,185 @@
+use crate::model::account;
+use crate::model::prelude::*;
+use async_std::stream::Stream;
+use iroha::{config::Configuration, crypto::KeyPair, torii::uri};
```

Comment:
I think it should be `iroha_client::config::Configuration`. The original client uses that.

Reply:
@eadventurous you're right. I was using an old version.